app/_components/GuessDistanceModal.tsx

```diff
@@ -48,7 +48,7 @@ const GuessDistanceModal = ({
     };
     try {
       const response = await fetch(
-        `${process.env.NEXT_PUBLIC_BASE_URL}/api/posthint`,
+        `${process.env.NEXT_PUBLIC_BASE_URL}/api/hints/post`,
```

Comment:
The method in the fetch options is already a `POST` so do we need to have `/post` in the url?